### PR TITLE
[full-ci] Show Legal statement by personal settings

### PR DESCRIPTION
**Pull request by T0mWz**
## Description
When your own legal statements are published, handy to show them at your personal settings page.

## How Has This Been Tested?
 - Applied within own environment

## Screenshots (if appropriate):
Screenshot 2022-11-23 at 10 51 48

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [X] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

## ToDo:
 - Translation option added

### CODE REVIEW

**Comment by mmattel** [comment]
@T0mWz you need to sign the CLA to proceed, see the link above, it is just a click...

**Comment by T0mWz** [comment]
ow, I already did that. But do it directly again @mmattel !

**Comment by mmattel** [comment]
> I already did that

hmmm, I just clicked on recheck but it is still the same...?
Can you pls redo signing the online CLA?

**Comment by phil-davis** [comment]
I rebased and force-pushed. Up-to-date CI is running.

The commit is by https://github.com/tomwezepoel @tomwezepoel and not by @T0mWz 

So "signing the CLA needs to be done by @tomwezepoel 

Tom - I suppose that both accounts are you, the same person? What happened?

**Comment by T0mWz** [comment]
> I rebased and force-pushed. Up-to-date CI is running.
> 
> The commit is by https://github.com/tomwezepoel @tomwezepoel and not by @T0mWz
> 
> So "signing the CLA needs to be done by @tomwezepoel
> 
> Tom - I suppose that both accounts are you, the same person? What happened?

Ahja, shared personality. Haha!

**Comment by mmattel** [comment]
A changelog entry is needed.

**Comment by mmattel** [comment]
@phil-davis @DeepDiver1975 approve?

File: settings/templates/panels/personal/profile.php [hunk 1]
```diff
@@ -150,3 +150,21 @@
...
+if (!empty($_['legal_privacy_policy']) || !empty($_['legal_imprint'])) {
+	?><div class="section">
+        <h2 class="legal"><?php p($l->t('Legal'));?></h2>
+        <?php if (!empty($_['legal_imprint'])) { ?>
+        <p>
+                <label for="legal_imprint"><?php p($l->t('Our imprint conditions:')); ?></label> <a href="<?php p($_['legal_imprint']) ?>"><?php p($_['legal_imprint']) ?></a>
+        </p>
+        <?php } if (!empty($_['legal_privacy_policy'])) { ?>
+        <p>
+                <label for="legal_privacy_policy"><?php p($l->t('Our Privacy Policy conditions:')); ?></label> <a href="<?php p($_['legal_privacy_policy']) ?>"><?php p($_['legal_privacy_policy']) ?></a>
```
Comment (DeepDiver1975) [inline settings/templates/panels/personal/profile.php:164]: I prefer to open the urls in a new tab - please add `target="_blank" rel="noopener noreferrer"`

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: settings/templates/panels/personal/profile.php (hunk 1 above)
Comment (T0mWz) [reply settings/templates/panels/personal/profile.php:164]: @DeepDiver1975  For both, privacy and imprint ?

Comment (DeepDiver1975) [reply settings/templates/panels/personal/profile.php:164]: Yes please - THX a lot!

**Comment by mmattel** [comment]
@T0mWz you need to run a php style fix. To do so, run 'make help' in the repo root and near the bottom of the output you will see the command example, then repush.

**Comment by phil-davis** [comment]
> A changelog entry is needed.

done

**Comment by phil-davis** [comment]
> @T0mWz you need to run a php style fix. To do so, run 'make help' in the repo root and near the bottom of the output you will see the command example, then repush.

done

**Comment by phil-davis** [comment]
Note: this makes use of the existing core settings legal.privacy_policy_url and legal.imprint_url that have existed for some time - e.g. see old PR https://github.com/owncloud/theme-enterprise/pull/25 and #31699

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@DeepDiver1975 or @jvillafanez please also review. IMO this is ready to merge.

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**